backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -92,11 +100,17 @@ export const createCampDtoValidator = async (
       }
     }
   }
-  if (req.body.campers) {
+  if (body.campers) {
     return res.status(400).send("campers should be empty");
   }
-  if (req.body.waitlist) {
+  if (body.waitlist) {
     return res.status(400).send("waitlist should be empty");
   }
+  if (req.file && !validateImageType(req.file.mimetype)) {
```

Comment:
super nitpicky but: can we change the variable name to something a bit more descriptive (e.g. campPhoto?)

Reply:
oh you mean like something like validateCampPhotoType?

Reply:
i meant `file`!

Reply:
adding custom properties to `Request` object would result in typescript throwing an [error](https://stackoverflow.com/questions/58957228/property-does-not-exist-on-type-requestparamsdictionary). In order to fix it, we'll need to add additional changes to `Request`'s type definition which could make the code more complex. Thus, sticking with property name `file`.